app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
```

Comment:
Consider adding a docstring or a custom message to provide more context for the `NotVaccinatedError` exception.